### PR TITLE
Add AccessibleEditableText classes

### PR DESCRIPTION
**Pull request by summer-alice**
This translates the Java versions of the `AccessibleEditableTextAdapter`, `AccessibleEditableTextEvent`, and `AccessibleEditableTextListener` classes to D.

The code is the same for both the win32 and Gtk versions.

### CODE REVIEW

File: org.eclipse.swt.gtk.linux.x86/src/org/eclipse/swt/accessibility/AccessibleEditableTextEvent.d [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+     * @see AccessibleEditableTextListener#replaceText
+     */
+    public int end;
+
+    /**
+     * [in] a string that will replace the specified character range
+     *
+     * @see AccessibleEditableTextListener#replaceText
+     */
+    public String string_;
```
Comment (jacob-carlborg) [inline org.eclipse.swt.gtk.linux.x86/src/org/eclipse/swt/accessibility/AccessibleEditableTextEvent.d:55]: Is this how it's done in other places in the code? `string` is not a keyword, just another symbol name.

Comment (summer-alice) [reply org.eclipse.swt.gtk.linux.x86/src/org/eclipse/swt/accessibility/AccessibleEditableTextEvent.d:55]: I couldn't (quickly) find any other occurrence of using `string` or `string_` as a field name. That said, there are times where `String string` is used (e.g. in [Label.setText]), so I'm happy to change it to `string` if that's preferred?

[Label.setText]: https://github.com/d-widget-toolkit/dwt/blob/202c52e/org.eclipse.swt.gtk.linux.x86/src/org/eclipse/swt/widgets/Label.d#L571

Comment (jacob-carlborg) [reply org.eclipse.swt.gtk.linux.x86/src/org/eclipse/swt/accessibility/AccessibleEditableTextEvent.d:55]: The preferred way is to aim for correctness first then the second goal is to be as close to the original Java source code (to make it easier to port new code in the future). It's go with `string`.

Comment (summer-alice) [reply org.eclipse.swt.gtk.linux.x86/src/org/eclipse/swt/accessibility/AccessibleEditableTextEvent.d:55]: That makes sense, I think you've said something similar before as well so I'll keep this in mind. I've updated the code.

**Review by jacob-carlborg — APPROVED** [review]

**Comment by jacob-carlborg** [comment]
Thanks.

**Comment by summer-alice** [comment]
> Thanks.

Thank you.

**Pull request merged**